_posts/2020-10-05-torch-network-with-autograd/torch-network-with-autograd.Rmd

```diff
@@ -0,0 +1,347 @@
+---
+title: "Introducing torch autograd"
+description: > 
+ Instructive though it may be to code backpropagation from scratch, with torch, there is hardly ever a reason to. Its automatic differentiation feature, called autograd, keeps track of operations that need their gradients computed, as well as how to compute them.
```

Comment:
I think for the description you should have it be as straightforward as possible. So, the first line might read something like: "With torch, there is hardly ever a reason to code backpropagation from scratch…"

Reply:
yeah, right, updating!